platform/android/export/export.cpp

```diff
@@ -824,10 +824,22 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 							encode_uint32(min_gles3 ? 0x00030000 : 0x00020000, &p_manifest.write[iofs + 16]);
 						}
 
+						if (tname == "meta-data" && attrname == "name") {
```

Comment:
It feels somewhat brittle to only check "meta-data" and "name" (and below "meta-data" and "value") and assume that it's the meta-data entry the XR mode. In the current template there is only one meta-data entry, but there's also room for some placeholders where more stuff can be injected, and I assume that there can be several meta-data entries in a valid manifest?

Would there be a way to actually retrieve the values of the "name" attribute to do this logic only when parsing the `xr_mode_metadata` meta-data entry?

Reply:
Done :)